src/test/java/org/jenkinsci/plugins/workflow/WorkflowTest.java

```diff
@@ -107,7 +107,9 @@ private void liveness() {
         assertNotNull(e);
         assertEquals(e, b.getExecutor());
         assertTrue(e.isActive());
+        /* TODO seems flaky:
```

Comment:
It is to avoid having an entire plugin build (for example, prior to release, or on an unrelated PR) fail due to a _rare_ error. But I saw this flake several times in a row. It looks like a race condition in the test. Until the test is improved to pass more reliably, I would rather suppress the flakiness.